model.py

```diff
@@ -3132,6 +3131,8 @@ def cover_thumbnail_url(self):
 
     @property
     def target_age_string(self):
+        if not self.target_age:
+            return None
```

Comment:
This needs to be the empty string, not None. There are a number of places where we append .target_age_string onto another string.